src/components/Integrations/BigQuery/FreeAccount/FreeAccount.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { BigQueryLogo } from './styles/styles';
+import { useIntl } from 'react-intl';
+import datastudiogif from '../../../../img/google-data-studio.gif';
+
+interface IFreeAccountProps {}
+
+const FreeAccount: React.FunctionComponent<IFreeAccountProps> = (props) => {
+  const intl = useIntl();
+  const _ = (id: string, values: any) => intl.formatMessage({ id: id }, values);
+
+  return (
+    <>
```

Comment:
No es necesario usar el <>, la idea de usar ese fragmento es cuando estas devolviendo varios elementos de react, pero aqui en este caso estás devolviendo uno solo (section). Igual no afecta el rendimiento ni nada por el estilo, pero no es necesario, se evita un nivel mas de anidamiento

Reply:
suggest accepted and applied